llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -934,8 +934,18 @@ void VPlan::prepareToExecute(Value *TripCountV, Value *VectorTripCountV,
 
   IRBuilder<> Builder(State.CFG.PrevBB->getTerminator());
   // FIXME: Model VF * UF computation completely in VPlan.
-  VFxUF.setUnderlyingValue(
-      createStepForVF(Builder, TripCountV->getType(), State.VF, State.UF));
+  if (VF.getNumUsers()) {
+    Value *RuntimeVF = getRuntimeVF(Builder, TripCountV->getType(), State.VF);
```

Comment:
nit: worth assigning `auto TripCountType = TripCountV->getType()` at the outset, or perhaps a shorter name.

Reply:
Done, thanks!